localstack/services/stepfunctions/asl/antlr/ASLIntrinsicLexer.g4

```diff
@@ -1,84 +1,59 @@
-lexer grammar ASLIntrinsicLexer;
+// $antlr-format alignTrailingComments true, columnLimit 150, maxEmptyLinesToKeep 1, reflowComments false, useTab false
```

Comment:
nice ✨  Are these known best practices for *.g4 files or did you come up with these rules?

Reply:
Yes, the same are used in the antlr4 grammar repository: https://github.com/antlr/grammars-v4